main.py

```diff
@@ -0,0 +1,148 @@
+import csv
+import logging
+
+
+class Account:
+    def __init__(self, name):
+        self.name = name
+        self.balance = 0
+        self.transactions = []
+
+    def add_money(self, amount):
+        self.balance += amount
+
+    def substract_money(self, amount):
+        self.balance -= amount
+
+    def add_transaction(self, date, narrative, amount, paid): # if paid is True -> person sent amount
+                                                               # if paid is False -> person received amount
+        self.transactions.append((date, narrative, amount, paid))
+        self.transactions.sort()
+
+    def print_transactions(self):
+        for transaction in self.transactions:
+            if transaction[3]:
+                print(self.name + " sent amount " + str(transaction[2]) + " on " + transaction[0] + "\nnarrative : " + transaction[1] + "\n")
+            else:
+                print(self.name + " received amount " + str(transaction[2]) + " on " + transaction[0] + "\nnarrative : " + transaction[1] + "\n")
+
+    def __str__(self):
+        if self.balance >= 0:
+            return "Name: " + self.name + "\nhas money : " + str(self.balance) + "\n"
+        else:
+            return "Name: " + self.name + "\nhas debt of : " + str(abs(self.balance)) + "\n"
+
+# Program start
+
+# Add logging
+logger = logging.getLogger(__name__)
+logging.basicConfig(filename='SupportBank.log', filemode='w', level=logging.DEBUG)
+
+logger.info("Starting to run SupportBank")
+
+accounts = []
+
+
+def find_account(acc_name):
+    for acc in accounts:
+        if acc.name == acc_name:
+            return acc
+    return False

```

Comment:
We could some list comprehension here 🎉 

You can try using [next](https://www.w3schools.com/python/ref_func_next.asp) and [filter](https://www.geeksforgeeks.org/filter-in-python/)